src/docs/product/performance/resources.md

```diff
@@ -0,0 +1,120 @@
+
+By default, the resources table is sorted by most time spent, which serves as a proxy for the relative performance impact of a given resource. A resource's time spent is the sum of all its durations in a given time period or, put another way, the product of its average duration and requests per minute. This means that resources at the top are usually loading really slowly, very frequently, or both.
+
+You can click on a column header to change how the table is sorted. Sort by requests per minute to see the most frequently loaded resources or by average duration to see the slowest-loading resources.
+
+To view more details, click on a resource from the table to open its **Resource Summary** page.
+
+### Resource Parameterization
+
+To enable Sentry to group similar resources together, Sentry parameterizes resource URLs, removing potenially dynamic elements. This helps track the performance of a particular resource across different releases, even when they have dynamic segments (used for busting caches or CDNS).
+
+To see an example of resource URL from a group, hover over a URL in the resource table.
+
+## Resource Summary Page
+
+To open a query's **Resource Summary** page, click on the resource from the table in the **Resources** page.
+
+At the top of the page, average encoded size, average decoded size, average transfer size, average duration, and requests per minute are shown for the selected time range. Right below this, you can see summary graphs for requests per minute, average duration, and average resource size over time.
+
+At the bottom, you can find a list of pages the resource is found in, sorted by the requests per minute for the resource on that page.
+
+If you want to investigate a specific page, click on it to open a sidebar showing some sample events.
+
+## Sample List
+
+Click on a page to open its samples list. This side panel shows summary metrics (requests per minute, average duration, and time spent) for the resource in the specific page.
+
+The table shows a sample list of events, their span duration, and the difference in duration compared to average. This table is sorted by longest span duration.
+
+Sentry automatically finds a variety of samples to help you investigate performance problems. The chosen events will cover the entire selected time range, as well as a range of durations (both faster than, slower than, and near the average duration). You can get a sense of the performance characteristics by comparing different sample events.
+
+These same events are also represented as triangles shown in the average duration graph above.
+
+You can generate a new list of random sample events by clicking the "Try Different Samples" button at the bottom.
+
+From the sample list, you can drill down to specific good, average, or bad examples of a given resource within a given page. Click on an event ID to drill into the resource's span details within the span waterfall of the **Event Details** page.
```

Comment:
```suggestion
From the sample list, you can drill down to specific good, average, or bad examples of a given resource within a given page. Click on an event ID to drill into the resource's span details within the span waterfall of the **Event Details** page.

<div style="position: relative; padding-bottom: calc(50.829268292682926% + 41px); height: 0;"><iframe src="https://demo.arcade.software/h2B547DqgJGIrlsMeVWx?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"></iframe></div>

```

Reply:
@vivianyentran I had a look at this arcade, there was an error in the sample sidebar here which should be fixed when this PR is merged https://github.com/getsentry/sentry/pull/60091

Reply:
Reffering to this error
<img width="731" alt="image" src="https://github.com/getsentry/sentry-docs/assets/44422760/d9e28063-772e-47d8-9734-6c170e22bb61">

Reply:
Yes, I noticed that too. I can replace the image / redo it once it gets fixed!

Reply:
@vivianyentran Just checked, that PR deployed, so we redo those images now